src/__test__/app.test.ts

```diff
@@ -0,0 +1,143 @@
+import Product from '../domain/Product';
+import VendingMachine from '../domain/VendingMachine';
+import { ERROR_MESSAGE } from '../constants';
+
+const addProduct = (name: string, price: number, quantity: number) => {
```

Comment:
```suggestion
const addProduct = ({name, price, quantity}: IProduct) => {
```
인자를 [객체형태](https://github.com/qkraudghgh/clean-code-javascript-ko#%ED%95%A8%EC%88%98functions)로 받아봅시다!~ 
테스트에 인자를 넣을때도 어떤 데이터인지 직관적으로 알 수 있어요!

Reply:
오오 interface를 이런식으로 활용할 수도 있었네요!!

Reply:
반영 링크입니다! [99dcb88](https://github.com/woowacourse/javascript-vendingmachine/pull/32/commits/99dcb880b05a9ecc828ce945bd8f1b390859fe57)